packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -126,17 +127,24 @@ class FloatingActionButton extends StatelessWidget {
     this.tooltip,
     this.foregroundColor,
     this.backgroundColor,
+    this.focusColor,
+    this.hoverColor,
     this.heroTag = const _DefaultHeroTag(),
     this.elevation,
+    this.focusElevation,
+    this.hoverElevation,
     this.highlightElevation,
     this.disabledElevation,
     @required this.onPressed,
     this.mini = false,
     this.shape,
-    this.clipBehavior = Clip.none,
+    this.clipBehavior,
```

Comment:
It doesn't change the default, just made it so that the `MaterialButton` was where the default was defined (since passing `null` selects the default of `Clip.none`).

However, I think I'll do both: select `Clip.none` if it's `null`, and also show it as the default as a form of documentation.